src/core/Toast/Toast.baseStyles.tsx

```diff
@@ -0,0 +1,42 @@
+import { css } from 'styled-components';
+import { SuomifiTheme } from '../theme';
+import { font, element } from '../theme/reset';
+
+export const baseStyles = (theme: SuomifiTheme) => css`
+  &.fi-toast {
```

Comment:
Is there a reason we have adopted this kind of style def for the top level? Not sure if it's bad, but it kind of serves no purpose as these classnames will change during the build => these styles cannot be overridden using `.fi-toast` selector if I'm not mistaken. So the top level styles, preferably only default positioning and size, should be easy to override. Probably a style wrapper is needed as well.

Reply:
At least there is no mutual conversation about to change to use this kind of approach. Same issue seems to be with the Notification, so should be fixed there also.

Reply:
Fixed changed to:
export const baseStyles = (theme: SuomifiTheme) => css`
  ${element(theme)}
  ${font(theme)('bodyTextSmall')}
  border-top: 4px solid ${theme.colors.successBase};
  width: 100%;
  box-shadow: ${theme.shadows.wideBoxShadow};
  border-radius: 4px;

  & .fi-toast-wrapper {

Reply:
Looks good now in my opinion.